samples/sampler/stories/qa/Panels.js

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable react/jsx-no-bind */
+import {handle, forward} from '@enact/core/handle';
+import kind from '@enact/core/kind';
+import {clamp} from '@enact/core/util';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select} from '@enact/storybook-utils/addons/knobs';
+import Routable, {Route, Linkable} from '@enact/ui/Routable';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import Header from '@enact/agate/Header';
+import Icon from '@enact/agate/Icon';
+import Item from '@enact/agate/Item';
+import {Panels, Panel, BreadcrumbPanels} from '@enact/agate/Panels';
+
+Panels.displayName = 'Panels';
+const Config = mergeComponentMetadata('Panels', Panels);
+
+const BasicPanels = (props) => {
+	const [index, setIndex] = React.useState(0);
+	const goNext = () => setIndex(clamp(0, 2, index + 1));
+	const goPrevious = () => setIndex(clamp(0, 2, index - 1));
+
+	return (
+		<Panels
+			index={index}
+			{...props}
+		>
+			<Panel>
+				<Header title="First Panel" />
+				<Item onClick={goNext}>Next</Item>
```

Comment:
Since there is only one menu here, it is difficult to know if the focus is preserved properly. 
Same menu, but add one more please
```suggestion
				<Item onClick={goNext}>Next1</Item>
				<Item onClick={goNext}>Next2</Item>
```

or 

```suggestion
				<Item onClick={goNext}>Pages</Item>
				<Item onClick={goNext}>General</Item>
```